test/commons/dom/is-visible.js

```diff
@@ -5,25 +5,24 @@ describe('dom.isVisible', function() {
   var queryFixture = axe.testUtils.queryFixture;
   var isIE11 = axe.testUtils.isIE11;
   var shadowSupported = axe.testUtils.shadowSupport.v1;
-  var fakeNode = {
-    nodeType: Node.ELEMENT_NODE,
-    nodeName: 'div'
-  };
+  var computedStyleStub;
 
   afterEach(function() {
     document.getElementById('fixture').innerHTML = '';
     axe._tree = undefined;
+
+    if (computedStyleStub) {
+      computedStyleStub.restore();
+      computedStyleStub = null;
+    }
   });
 
   describe('default usage', function() {
     // Firefox returns `null` if accessed inside a hidden iframe
     it('should return false if computedStyle return null for whatever reason', function() {
-      var orig = window.getComputedStyle;
-      window.getComputedStyle = function() {
-        return null;
-      };
-      assert.isFalse(axe.commons.dom.isVisible(fakeNode));
-      window.getComputedStyle = orig;
+      computedStyleStub = sinon.stub(window, 'getComputedStyle').returns(null);
```

Comment:
If this test failed it would not reset the override and so all future tests would fail.